lib/productions/callback-interface.js

```diff
@@ -16,8 +20,9 @@ export class CallbackInterface extends Container {
       tokeniser,
       new CallbackInterface({ source: tokeniser.source, tokens }),
       {
-        inheritable: !partial,
+        inheritable: !!inheritable,
```

Comment:
Can we make this change separately? We should either change all `partial` to `inheritable` everywhere or keep it everywhere, and making that change here sounds like out-of scope.

Reply:
Wasn't the issue here that callback interface doesn't support `partial` at all?

Reply:
Oh hmm, I think we talked about it somewhere but can't find it anywhere, do you remember? I now wonder why we need it at all, neither the spec nor the mozilla-central uses inheritance for callback interfaces, right?

Reply:
Ah I found it, and I think we should just remove `inheritable`s here.